src/aap_eda/api/serializers/project.py

```diff
@@ -126,7 +124,6 @@ def to_representation(self, project):
             "credential": credential,
             "import_state": project["import_state"],
             "import_error": project["import_error"],
-            "import_task_id": project["import_task_id"],
```

Comment:
@Alex-Izquierdo It is removed here. You might looking at the wrong branch, maybe?